geniza/corpus/management/commands/add_links.py

```diff
@@ -255,32 +272,82 @@ def add_link(self, row):
         doc = Document.get_by_any_pgpid(int(row["object_id"]))
         if not doc:
             self.stats["document_not_found"] += 1
+            self.not_found_documents += [row]
             if self.verbosity > self.v_normal:
                 self.stdout.write(
                     "Document %s not found in database" % row["object_id"]
                 )
             return
 
         # target url: combine base url for this link type with link target from csv
-        target_url = "".join([self.base_url[link_type], row["link_target"]])
+        target_url = "".join([self.base_url[link_type], quote(row["link_target"])])
```

Comment:
this looks correct to me, do we have any tests that confirm ?

Reply:
Do you think this is sufficient, or should it be tested more thoroughly?
https://github.com/Princeton-CDH/geniza/blob/d90f60f1ce79b7d244f130f2f285e1d74a1b02c0/geniza/corpus/tests/test_add_links.py#L316-L323

Reply:
perfect, thank you! sorry I didn't look more closely and find this myself

Reply:
No problem! I actually amended it to manually use the URL encoding rather than use the `quote` function again, so I'm glad you brought it to my attention :)